### ISSUE TITLE
pe-layout sensitivity of BGCEXP_LNDATM_CNPRDCTC tests

### ISSUE THREAD

**Issue by wlin7** [E3SM-Project/E3SM#5992]
SMS_Ln5.ne30pg2_r05_EC30to60E2r2.BGCEXP_LNDATM_CNPRDCTC_1850 and SMS_Ln5.ne30pg2_r05_EC30to60E2r2.BGCEXP_LNDATM_CNPRDCTC_20TR tests appear to be
sensitive PE-layout changes. 

Tests on chrysalis created specifying --pesfile config_pes_tests.xml produced different results from tests not specifying --pesfile.
The former uses P128x2 (with threading), while the latter uses staggered 22-node layout without threading.

The difference could be simply due to build with SMP or not.

**Comment by rljacob** [comment]
Yes we already know that compiling with and without openmp produces bfb differences.
